src/liballoc/str.rs

```diff
@@ -27,7 +27,7 @@
 //! We can explicitly specify `hello_world`'s lifetime as well:
 //!
 //! ```
-//! let hello_world: &'static str = "Hello, world!";
```

Comment:
this is in documentation showing how to explicitly specify a lifetime, so it shouldn't be removed

Reply:
Yes, can this be reverted?

Reply:
Done